src/main/java/Task.java

```diff
@@ -0,0 +1,10 @@
+public class Task {
+    public int status;
```

Comment:
Does the status(0, 1) refer to whether or not the task is done? If that's the case, changing it to a boolean(eg. isDone) will be helpful in terms of readability. It could help to reduce confusion as your teammates will not have to guess which variable does what when they read your code.